dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -186,9 +186,9 @@ BEGIN
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Compatibilité", -1, 7, 67, 285, 97
     LTEXT "Quelques programmes peuvent mal fonctionner si vous ne redémarrez pas votre ordinateur après que vous ayiez changé les paramètres des couleurs.\n\nAprès que j'ai changé les paramètres des couleurs", -1, 12, 78, 270, 40
-    AUTORADIOBUTTON "&Redémarrer l'ordinateur avant d'appliquer les nouveaux paramètres des couleurs.", IDC_RESTART_RB, 14, 116, 273, 10
-    AUTORADIOBUTTON "Appliquer les nouveaux paramètres des couleurs sans redémarrer.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Me demander avant d'appliquer les nouveaux paramètres des couleurs.", IDC_ASKME_RB, 14, 146, 243, 10
+    AUTORADIOBUTTON "&Redémarrer l'ordinateur avant d'appliquer les nouveaux paramètres des couleurs", IDC_RESTART_RB, 14, 116, 273, 10
+    AUTORADIOBUTTON "Appliquer les nouveaux paramètres des couleurs sans redémarrer", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10//FIXME: add accel
+    AUTORADIOBUTTON "Me demander avant d'appliquer les nouveaux paramètres des couleurs", IDC_ASKME_RB, 14, 146, 243, 10//FIXME: add accel
```

Comment:
```suggestion
    AUTORADIOBUTTON "Me &demander avant d'appliquer les nouveaux paramètres des couleurs", IDC_ASKME_RB, 14, 146, 243, 10
```
I had to use a different accelerator here since we have the sentence reformulated here compared to on Windows.

By the way, the "des couleurs" isn't present there (and in english these are "display settings", i.e. "paramètres d'affichage")
![image](https://github.com/user-attachments/assets/a8ab9415-0cea-43a5-886e-ecad872d9c58)

Reply:
I decided to do what Windows does. Thank you for the screenshot. I oriented on this one ultimately for french. Resolved.